repos.d/managarm.yaml

```diff
@@ -0,0 +1,28 @@
+###########################################################################
+# Managarm
+###########################################################################
+
+- name: managarm_x86_64
+  type: repository
+  desc: Managarm x86_64
+  family: managarm
```

Comment:
You need to add the family to repology-schemacheck.py as well.
Add `ruleset: managarm` here please too, as the plan is to make these explicitly separate (`family` is used to group related repositories, and `ruleset` is used to create specific rules).